### DISCUSSION TITLE
Is there a way to prevent installation of `.def` files?

### DISCUSSION THREAD

**Discussion by realmazharhussain** [lu-zero/cargo-c#319] [Q&A]
I packaged `gst-plugins-rs` for msys2. The options I use during build/install are `--workspace --offline --release --library-type=cdylib --prefix="${MINGW_PREFIX}" --exclude gst-plugin-csound`. This installs `.def` files along with the `.dll` files.

I can easily remove all the `.def` files after `cinstall` has done its thing. But I was wondering if there is any option I can use to prevent `.def` files from getting installed in the first place?

**Comment by lu-zero** [comment]
Being plugins probably you may omit the .def generation as you omit the headers generation. @sdroege do you have opinions in this regard?

**Reply by realmazharhussain** [reply]
My question was more about how?

I don't have any experience developing in rust. So, I don't know how would I omit .def generation.

**Reply by lu-zero** [reply]
We could add a configuration item to enable/disable the .def generation like we do for the headers since they are logically similar. I wanted @sdroege since he would potentially use the feature in `gst`.

**Reply by realmazharhussain** [reply]
Got it. Thanks.

**Comment by sdroege** [comment]
I don't think it makes sense to ship the `.def` file. We don't include them in GStreamer for plugins.
